src/app/views/rolodex_view.js

```diff
@@ -1,6 +1,83 @@
+import $ from 'jquery';
+import _ from 'underscore';
 import Backbone from 'backbone';
+import ContactView from 'app/views/contact_view';
+import ContactDetailView from 'app/views/contact_detail_view';
 
 const RolodexView = Backbone.View.extend({
+  initialize: function(options){
+    this.input = options.input;
+
+    this.cardList = [];//create empty array variable.
+
+    //loop through collection of models and add contacts to cardList
+    this.model.forEach(function(contact) {
+      this.addContact(contact);
+    }, this);
+
+    this.listenTo(this.model, "update", this.render);
+
+    this.listenTo(this.model, "add", this.addContact);
+
+    this.listenTo(this.model, "remove", this.removeContact);
+
+  },
+
+  render: function() {
+    $('#contact-cards').empty();//empties every time rerendered; elements are bound in contact-vew
+    // $('#contact-cards').empty(); <----things were weird when I ran this. Still not sure why this is unneccessary. Is it because of the listenTo update above?
+    //loops through cardList and appends each card to correct tag in index.html
+    this.cardList.forEach(function(card) {
+      card.render();
+      $('#contact-cards').append(card.$el);
+    }, this);
+
+    // Enable chained calls
+    return this;
+  },
+
+  makeDetail: function(model){
+    $('#contact-details').empty();//clears modal of previous details
+    //create new detail view
+    var cardDetail = new ContactDetailView({
+      model: model
+    });
+    cardDetail.render(); //render the details
+    this.listenTo(cardDetail, "editMe", this.editCard);
+    $('#contact-details').append(cardDetail.$el); //append card details
+    $('#contact-details').show(); //show modal
+  },
+
+  editCard: function(cardModel) {
+  console.log("Editing a card");
+
+  this.input.name.val(cardModel.get("name"));
+  this.input.email.val(cardModel.get("email"));
+  this.input.phone.val(cardModel.get("phone"));
+  this.model.remove(cardModel); //temporarily remove contact from collection while edited
+},
+
+  addContact: function(contact) {
+    var card = new ContactView({
+      // el: $('#contact-cards'), <---still unsure why when el was specified (and direct li objects were added to ul tag), .contact-card tags could not be clicked independently
```

Comment:
Because when el was specified each view was encompassing the same area on the DOM so any event was triggering all their listeners.